src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/controller/InterventionController.kt

```diff
@@ -0,0 +1,25 @@
+package uk.gov.justice.digital.hmpps.hmppsinterventionsservice.controller
+
+import org.springframework.http.HttpStatus
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.PathVariable
+import org.springframework.web.bind.annotation.RestController
+import org.springframework.web.server.ResponseStatusException
+import uk.gov.justice.digital.hmpps.hmppsinterventionsservice.dto.InterventionDTO
+import uk.gov.justice.digital.hmpps.hmppsinterventionsservice.service.InterventionService
+import uk.gov.justice.digital.hmpps.hmppsinterventionsservice.utils.StringToUUIDConverter
+
+@RestController
+class InterventionController(
+  private val interventionService: InterventionService,
+) {
+
+  @GetMapping("/intervention/{id}")
```

Comment:
@sldblog do we want the endpoints to differentiate between structured and DF interventions?

Reply:
Given we're expected to support discovering all interventions and services, I think we're currently ok with a common prefix.

These are implied to be published or available interventions (vs in-progress, unlisted, under QA) -- what do you think about using `/published-interventions/{id}`?

Reply:
hmm...good point - maybe we should discuss this to decide